conf/application.conf

```diff
@@ -255,17 +278,64 @@ assets {
 
 agentAssuranceFlag=false
 
-contact-frontend {
-  host = "http://localhost:9250"
-}
+contact-frontend.host = "http://localhost:9250/contact/problem_reports_"
+
+reportAProblemPartialUrl = ${contact-frontend.host}"ajax?service="${appName}
+reportAProblemNonJSUrl = ${contact-frontend.host}"nonjs?service="${appName}
 
 authentication {
   login-callback.url = "http://localhost:9437"
   government-gateway.sign-in.path = "/gg/sign-in"
   government-gateway.sign-in.base-url = "http://localhost:9025"
 }
+
 government-gateway.url = "http://www.ref.gateway.gov.uk/"
 
 continueUrl {
   domainWhiteList = ["localhost","www.tax.service.gov.uk"]
 }
+
+microservice {
+  metrics {
+    graphite {
+      host = localhost
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
+      legacy = false
+    }
+  }
+
+  services {
+    auth {
+      host = localhost
+      port = 8500
+    }
+    agent-subscription {
+      host = localhost
+      port = 9001
+      someInt = 1000
+      someString = "someString"
+      someBoolean = true
+    }
+  }
+}
+
+controllers {
+  com.kenshoo.play.metrics.MetricsController = {
```

Comment:
This `com.kenshoo.play.metrics.MetricsController` block is already defined in the newly included `frontend.conf` - https://github.com/hmrc/bootstrap-play-25/blob/master/src/main/resources/frontend.conf